src/native/minipal/random.h

```diff
@@ -3,8 +3,15 @@
 
 #pragma once
 
-#include "pal_compiler.h"
-#include "pal_types.h"
+#include <libs/Common/pal_compiler.h>
```

Comment:
We should avoid including code from outside the minipal in the minipal. The minipal shouldn't have dependencies on any other components in the repo, as those other components likely have dependencies on the minipal.